### ISSUE TITLE
3D model with roughness under 0.05, does not work on iPhone 7

### ISSUE THREAD

**Issue by dam00n** [viromedia/viro#673]
- [ ] Review the documentation: https://docs.viromedia.com/
  - [ ] Search for existing issues: https://github.com/viromedia/viro
  - [ ] Use the latest ViroReact release: https://docs.viromedia.com/docs/releases

## Environment
Please provide the following information about your environment:
1. Development OS: Mac

2. Device OS & Version: iOS version

3. Version: ViroReact 2.15.0

4. Device(s): iPhone 7

## Description
Using 2.15.0, the roughness under 0.05 still does not work on iPhone 7. When roughness factor <= 0.05 or if the value in a roughness texture <= 0.05, the material still appears all black (and also breaks the depth buffer actually, as it always appear on top of everything).

It may be worth noting that while using the gradient texture makes the material appear fully black on the edge where the values are under 0.05, a fully black roughness texture successfully makes the material fully glossy even on iPhone 7.

**Comment by silvainSayduck** [comment]
I've been trying to get my hands on more iOS models to check the status, but so far, I can only report:

- iPhone 7 (iOS 12 & 13): not working
- iPhone 8 (iOS 12): working

**Comment by silvainSayduck** [comment]
Any chance this has been fixed in 2.16.0 @dam00n ?

**Comment by radvani** [comment]
Yes, this should be fixed in 2.16.0. Let us know if you still have issues. Thanks!
